packages/backend/src/controllers/exploreController.ts

```diff
@@ -189,11 +180,11 @@ export class ExploreController extends BaseController {
                 csvLimit,
                 showTableNames,
                 customLabels,
-                columnOrder,
-                hiddenFields,
                 chartName: body.chartName,
                 fromSavedChart: false,
-                pivotConfig: csvPivotConfig,
+                columnOrder,
```

Comment:
columnOrder and hiddenFields are included in pivotConfig, but since pivtoConfig is optional, we use it separately, as it is needed for non pivoted tables.